common/src/main/java/org/schemarepo/config/Config.java

```diff
@@ -90,6 +105,9 @@
 
     DEFAULTS.setProperty(DEFAULT_SUBJECT_VALIDATORS,"");
 
+    // Validation defaults. The read strategy class is "Always Fail" unless configured.
+    DEFAULTS.setProperty(VALIDATION_READ_STRATEGY_CLASS, "org.schemarepo.validation.AlwaysFailValidationStrategy");
```

Comment:
No, that's not it at all. The 'Strategy' only goes with a validator through the '*.strategy.class' configuration. The 'CanRead' strategy is used by the default 'CanBeRead' and 'MutualRead' strategy (because why implement separately, one is just the reverse of the other, or combination of both), This setting to configures the 'CanRead' strategy to use in others that build off of it. 

If you attempt to use 'CanBeRead' without configuring this 'CanRead' then you will fail all registrations.